src/plugins/justwatch/hub.ts

```diff
@@ -0,0 +1,373 @@
+    shortDescription
+    interactions {
+      likelistAdditions
+      dislikelistAdditions
+      __typename
+    }
+    scoring {
+      imdbVotes
+      imdbScore
+      tmdbPopularity
+      tmdbScore
+      tomatoMeter
+      certifiedFresh
+      jwRating
+      __typename
+    }
+    interactions {
+      votesNumber
+      __typename
+    }
+    dailymotionClips: clips(providers: [DAILYMOTION]) {
+      sourceUrl
+      externalId
+      provider
+      streamUrl
+      __typename
+    }
+    posterUrl(profile: $profile, format: $format)
+    ... on MovieOrShowOrSeasonContent {
+      backdrops(profile: $backdropProfile, format: $format) {
+        backdropUrl
+        __typename
+      }
+      __typename
+    }
+    isReleased
+    credits(role: $creditsRole) {
+      name
+      personId
+      __typename
+    }
+    runtime
+    genres {
+      translation(language: $language)
+      shortName
+      __typename
+    }
+    __typename
+  }
+  likelistEntry {
+    createdAt
+    __typename
+  }
+  dislikelistEntry {
+    createdAt
+    __typename
+  }
+  watchlistEntryV2 {
+    createdAt
+    __typename
+  }
+  customlistEntries {
+    createdAt
+    __typename
+  }
+  freeOffersCount: offerCount(
+    country: $country
+    platform: WEB
+    filter: {monetizationTypes: [FREE, ADS]}
+  )
+  watchNowOffer(country: $country, platform: WEB, filter: $watchNowFilter) {
+    ...WatchNowOffer
+    __typename
+  }
+  ... on Movie {
+    seenlistEntry {
+      createdAt
+      __typename
+    }
+    __typename
+  }
+  ... on Show {
+    tvShowTrackingEntry {
+      createdAt
+      __typename
+    }
+    seenState(country: $country) {
+      seenEpisodeCount
+      progress
+      __typename
+    }
+    __typename
+  }
+}
+
+fragment WatchNowOffer on Offer {
+  __typename
+  id
+  standardWebURL
+  preAffiliatedStandardWebURL
+  streamUrl
+  package {
+    id
+    icon
+    packageId
+    clearName
+    shortName
+    technicalName
+    iconWide(profile: S160)
+    hasRectangularIcon(country: $country, platform: WEB)
+    __typename
+  }
+  retailPrice(language: $language)
+  retailPriceValue
+  lastChangeRetailPriceValue
+  currency
+  presentationType
+  monetizationType
+  availableTo
+  dateCreated
+  newElementCount
+}`;
+
+type JustWatchHubChunk = PseuplexFeedHubChunk<JustWatchTitle, void, string>;
+
+export class JustWatchHub extends PseuplexFeedHub<JustWatchTitle, void, string, JustWatchHubOptions> {
+	private _totalItemsFetched: number = 0;
+	
+	constructor(options: JustWatchHubOptions) {
+		super(options);
+	}
+
+	get metadataTransformOptions(): PseuplexMetadataTransformOptions {
+		return {
+			metadataBasePath: '/library/metadata',
+			qualifiedMetadataId: true
+		};
+	}
+
+	override parseItemTokenParam(itemToken: string | number): void | undefined {
+		// JustWatch doesn't use item tokens for pagination
+		return undefined;
+	}
+
+	override compareItemTokens(itemToken1: void, itemToken2: void) {
+		return 0;
```

Comment:
This makes items always match each other, which is probably not what you want. The purpose of it is for hubs that only ever add new items to the beginning (ie: a feed), so that it knows when to stop loading. If the list could be different every time, you wanna just do [this](https://github.com/lufinkey/pseuplex/blob/16e18dc66eff59a2c8451b5c4b336eb9a79e733f/src/plugins/letterboxd/filmshub.ts#L42) to force it to load the whole thing (hacky thing i did at the time because i didn't want to make another base class, but its fine)